Pocket/pocket-ios
# feat(remove) Remove the URL paste to add functionality

## Summary
Removed the ability to paste a URL through a banner

## References 
https://getpocket.atlassian.net/browse/IN-1333

## Implementation Details
Removed code and UI

## Test Steps
Copy a URL to the device pasteboard, open the app and you should not see the banner

## PR Checklist:
